src/AccessibilityInsights.Extensions.AzureDevOps/ConnectionCache.cs

```diff
@@ -78,7 +78,11 @@ private bool AddToCachePrivate(ConnectionInfo connection)
             bool reachedCapacity = CachedConnections.Count >= CAPACITY;
             if (reachedCapacity)
             {
-                CachedConnections.OrderBy(c => c.LastUsage);
```

Comment:
The analyzer pointed out that OrderBy doesn't impact the original input. The intention here was to sort the list, so I changed it to Sort. That then made me realize that the unit test wasn't covering the case where the list was unsorted, so I rewrote the unit test to handle it.